projects/forge-lib/src/lib/shared/form-components/concrete/currency/currency.component.html

```diff
@@ -0,0 +1,30 @@
+<div flex-container-row>
+  <button [id]="'prefix_' + id" class="prefix-suffix" sws-button-simple--jet m-t-25 m-r-0
+    *ngIf="display.prefix">{{display.prefix}}</button>
+  <div sws-form input-active full-width>
+    <input [id]="'input_' + id" forgeNumber [requireDecimal]="true" [decimalPlaces]="2" [(ngModel)]="value"
+      sws-form__input type="text" #name="ngModel" [name]="'input_' + id" [disabled]="display.disabled"
+      [required]="validation.required" onkeyup="this.setAttribute('value', this.value);">
```

Comment:
you shouldnt need the "this" context in the HTML

Reply:
I believe this is used when we have a directive to manipulate the DOM. I was following the structure of other components that were already developed and they used the same onkeyup="this.setAttribute...."